api/core/v1beta2/clusterclass_types.go

```diff
@@ -81,14 +81,15 @@ type ClusterClass struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	// spec is the desired state of ClusterClass.
-	// +optional
-	Spec ClusterClassSpec `json:"spec,omitempty"`
+	// +required
```

Comment:
required because spec is required in general (CC won't work if it's entirely empty, we have some work in general to do to improve validation for the fields in spec)